tests/libcxx/CMakeLists.txt

```diff
@@ -109,5 +124,12 @@ foreach(testcase ${alltests})
         endif()
     endif()
 
+    # Skip the following tests on GCC version less than 7 because a bug is not fixed or C++17 features are not supported.
+    if ("${name}" IN_LIST GCC_LESS_7_UNSUPPORTED_LIST OR "${testcase}" IN_LIST CXX_17_TEST_LIST)
```

Comment:
I am wondering thy the OR is needed here. Should all C++17 tests be skipped, or is skipping the ones in GCC_LESS_7_UNSUPPORTED_LIST  enought?

Reply:
Unfortunately these two tests are not overlapped.
Tests in `GCC_LESS_7_UNSUPPORTED_LIST` does not depend on C++17 features, but a bug that is resolved in GCC 7 (i.e., work with gcc 7 and c++14).
`CXX_17_TEST_LIST` is a specific set of tests to C++17 features only.